src/Servers/HttpSys/test/FunctionalTests/Http3SupportedAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Quic;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.Win32;
+
+namespace Microsoft.AspNetCore.Server.HttpSys
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = false)]
+    public class Http3SupportedAttribute : Attribute, ITestCondition
```

Comment:
I don't see anything that uses this attribute.

Reply:
Never mind, I searched for Http3SupportedAttribute but there is `[Http3Supported]`.